racetime/models/race.py

```diff
@@ -81,6 +81,12 @@ class Race(models.Model):
         related_name='opened_races',
         null=True,
     )
+    opened_by_bot = models.CharField(
```

Comment:
This field should be added to the `readonly_fields` list in admin: https://github.com/racetimeGG/racetime-app/blob/master/racetime/admin/__init__.py#L147-L155

Reply:
added